README.md

```diff
@@ -110,8 +103,19 @@ branch:
 #### Updating the name servers
 
 The primary way to access the DAO dashboard is through the `api3.eth` ENS name, which points directly to the IPFS hash.
-Then, the user can either connect to mainnet on their Metamask and visit `api3.eth/` (the recommended way), or they can visit `https://api3.eth.link/` (this is reported to be down frequently, see [this](https://blog.cloudflare.com/cloudflare-distributed-web-resolver/) for more information).
-We have also forwarded `https://dao.api3.org` to the IPFS hash (using the `dweb.link` gateway), but we do not recommend using this unless necessary.
+Then, the user can either connect to mainnet on their Metamask and visit `api3.eth/` (the recommended way), or they can
+visit `https://api3.eth.link/`.
+
+<!-- markdown-link-check-disable -->
+<!-- The link below exists and works, but the github actions check says it does not" -->
```

Comment:
I wonder why these links fail 😕

Reply:
Not sure, but some fail all the time and some are extremely flaky, so I've ignored those ones.